### ISSUE TITLE
Full builder image can't be pulled in a KinD cluster

### ISSUE THREAD

**Issue by ImJasonH** [paketo-buildpacks/full-builder#191]
## What happened?

* What were you attempting to do?

I originally found this trying to use `paketobuildpacks/builder:full` with Tekton on KinD, but I can reproduce it without Tekton, just by creating a Pod on a standard KinD cluster (via `kind create cluster`)

```
apiVersion: v1
kind: Pod
metadata:
  name: test-pod
spec:
  containers:
  - name: test
    image: docker.io/paketobuildpacks/builder:full
```

* What did you expect to happen?

I expect the image to be pulled and run as normal.

* What was the actual behavior? Please provide log output, if possible.

```
  Warning  Failed     3m10s                 kubelet            Failed to pull image "docker.io/paketobuildpacks/builder:full": rpc error: code = InvalidArgument desc = failed to pull and unpack image "docker.io/paketobuildpacks/builder:full": failed to prepare extraction snapshot "extract-106083700-MYlB sha256:837d6facb613e572926fbfe8cd7171ddf5919c1454cf4d5b4e78f3d2a7729000": info.Labels: label key and value greater than maximum size (4096 bytes), key: containerd: invalid argument
  Warning  Failed     3m10s                 kubelet            Error: ErrImagePull
  Normal   BackOff    3m7s (x2 over 3m10s)  kubelet            Back-off pulling image "docker.io/paketobuildpacks/builder:full"
  Warning  Failed     3m7s (x2 over 3m10s)  kubelet            Error: ImagePullBackOff
  Normal   BackOff    3m7s                  kubelet            Back-off pulling image "docker.io/paketobuildpacks/builder:full"
  Warning  Failed     3m7s                  kubelet            Error: ImagePullBackOff
```

The config file for the image is 23KB, including 22KB of labels alone.

## Build Configuration

* What platform (`pack`, `kpack`, `tekton` buildpacks plugin, etc.) are you
  using? Please include a version.

I'm using KinD.

I don't know if this reproduces on other cluster setups using containerd, or if it's something KinD can be configured to support.

* What buildpacks are you using? Please include versions.

paketobuildpacks/builder:full, currently sha256:badf0ce37e943707156af0a3b6dd874acf5545e15bb0eb4598b99468c429e59e

* What builder are you using? If custom, can you provide the output from `pack
  inspect-builder <builder>`?

paketobuildpacks/builder:full

* Can you provide a sample app or relevant configuration (`buildpack.yml`,
  `nginx.conf`, etc.)?

N/A

## Checklist

* [y] I have included log output.
* [y] The log output includes an error message.
* [y] I have included steps for reproduction.

**Comment by ryanmoran** [comment]
I believe this is an issue with containerd: https://github.com/containerd/containerd/issues/4684. It appears that this is fixed in v1.5.0 (which is still in beta), but was also backported to [1.4.2](https://github.com/containerd/containerd/releases/tag/v1.4.2). Can you try this with a newer version of containerd?

**Comment by ImJasonH** [comment]
Oh nice. 

I ran `brew upgrade kind` and re-created the cluster and the Pod works now. 🎉
